cmd/observer/node_builder/observer_builder.go

```diff
@@ -473,7 +473,13 @@ func (builder *ObserverServiceBuilder) BuildExecutionDataRequester() *ObserverSe
 		}).
 		Component("execution data service", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
 			var err error
-			bs, err = node.Network.RegisterBlobService(channels.ExecutionDataService, ds)
+			bs, err = node.Network.RegisterBlobService(channels.ExecutionDataService, ds,
+				p2p.WithBitswapOptions(
+					bitswap.WithTracer(
+						p2p.NewTracer(node.Logger.With().Str("blob_service", channels.ExecutionDataService).Logger()),
+					),
+				),
```

Comment:
Are these modules correct?
```suggestion
				blob.WithBitswapOptions(
					bitswap.WithTracer(
						blob.NewTracer(node.Logger.With().Str("blob_service", channels.ExecutionDataService).Logger()),
					),
				),
```